### PR TITLE
Add browse-pvc manifest

### PR DESCRIPTION
**Pull request by clbx**
Hello, I'd like to add my plugin browse-pvc to the krew repo. I constantly found myself spinning up dummy pods to exec into them so I could browse a PVC. I wrote this so I can easily, from the command line, easily get a shell with the PVC mounted. 

I don't think there's something similar to this already in the repo. Please feel free to leave any feedback.

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/browse-pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Krew Manifest
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: browse-pvc
+spec:
+  version: "v0.3.0"
+  homepage: https://github.com/clbx/kubectl-browse-pvc
+  shortDescription: "Browse PVCs from the command line"
+  description: "Browse PVCs from the command line"
```
Comment (ahmetb) [inline plugins/browse-pvc.yaml:10]: You can add a bit more detail here as to how the plugin does this task.

File: plugins/browse-pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  platforms:
+  # macOS
+  - selector:
+      matchLabels:
+        os: darwin
+    uri: https://github.com/clbx/kubectl-browse-pvc/releases/download/v0.3/kubectl-browse-pvc-darwin.zip
+    sha256: "0e0fc4e163e8434f9142edaa3b41fb1246e5c4dc652947284d6b8d49334601bd"
+    files:
```
Comment (ahmetb) [inline plugins/browse-pvc.yaml:18]: You can omit this field (as this value is the default)

Comment (clbx) [reply plugins/browse-pvc.yaml]: Removed

File: plugins/browse-pvc.yaml (hunk 1 above)
Comment (clbx) [reply plugins/browse-pvc.yaml]: Added

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**